libmscore/stafftypechange.cpp

```diff
@@ -108,9 +108,24 @@ void StaffTypeChange::draw(QPainter* painter) const
          lw, Qt::SolidLine, Qt::SquareCap, Qt::MiterJoin));
       painter->setBrush(Qt::NoBrush);
       painter->drawRect(0, 0, w, h);
-      QFont f("FreeSans", 12.0 * _spatium * MScore::pixelRatio / SPATIUM20);
-      painter->setFont(f);
-      painter->drawText(QRectF(0.0, 0.0, w, h), Qt::AlignCenter, QString("S"));
+
+      // draw simple preview icon
+      int lines = 5;
+      if (staffType()) {
+            if (staffType()->stemless())
+                  drawSymbol(SymId::noteheadBlack, painter, QPoint(w * 0.5 - 0.33 * _spatium, h * 0.5), 0.5);
```

Comment:
do we really care about spatium here? Why not use a fixed line distance?

Reply:
Well, i'm not sure, but the square h and w is calculated with spatium since stf was implemented (says Blame). I didn't change that, just got rid of the S and inserted the lines.